src/config/app.js

```diff
@@ -1,5 +1,12 @@
-import pkg from "../../package.json" with { type: "json" };
+import { createRequire } from "module";
+import { fileURLToPath } from "url";
+import path from "path";
 
+const __filename = fileURLToPath(import.meta.url);
```

Comment:
I imagine this is going to be a very common pattern we'll reuse in a few places. It might be a good idea to split this into a helper.